Makefile

```
@@ -0,0 +1,152 @@
+    ifeq ($(UNAME_M),arm64)
+        RID = osx-arm64
+    else
+        RID = osx-x64
+    endif
+else ifeq ($(OS),Windows_NT)
+    RID = win-x64
+else
+    RID = linux-x64
+endif
+
+.PHONY: help build build-oracle test clean install
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)